### ISSUE TITLE
Review docstrings of argument models

### ISSUE THREAD

**Issue by tcompa** [fractal-analytics-platform/fractal-tasks-core#421]
I'll merge #410 without polishing docstrings of the inner models. We should then review their format in view of:
- How they render in the documentation
- #418

(this also holds for https://github.com/fractal-analytics-platform/fractal-tasks-core/issues/414, https://github.com/fractal-analytics-platform/fractal-tasks-core/issues/412, https://github.com/fractal-analytics-platform/fractal-tasks-core/issues/404, https://github.com/fractal-analytics-platform/fractal-tasks-core/issues/401)

**Comment by tcompa** [comment]
Note that the actual docstring of a custom Pydantic model should likely never be shown in fractal-web, as described in https://github.com/fractal-analytics-platform/fractal-web/issues/198. Therefore we can keep it in a verbose form, since this is mostly useful as part of the documentation.

The attribute descriptions, however, _will_ be shown.

**Comment by tcompa** [comment]
There are known glitches in how these descriptions are shown in the documentation, but I'm starting to think that we should proceed e.g. with #397 and revamp the whole documentation.

**Comment by tcompa** [comment]
> As part of
> * #436 
> 
> we also started reviewing attribute descriptions for custom task-arguments Pydantic models (since they will now become part of the JSON Schemas). As soon as it is merged, it would be useful to have a second look at them.

This is now merged, with the new manifest available at https://fractal-analytics-platform.github.io/fractal-tasks-core/manifest.html.

@jluethi would you like to review the new (minimalistic..) attribute docstrings? The relevant classes are the ones listed in
```
    "OmeroChannel": "lib_channels.py",
    "Window": "lib_channels.py",
    "Channel": "lib_input_models.py",
    "NapariWorkflowsInput": "lib_input_models.py",
    "NapariWorkflowsOutput": "lib_input_models.py",
```

Other than that, this issue can be closed as it only concerns making the documentation prettier.

**Comment by tcompa** [comment]
> This is now merged

And available in 0.10.0a6

**Comment by jluethi** [comment]
I'll review those docstrings, thanks @tcompa

**Comment by tcompa** [comment]
Closed with #440.
